### ISSUE TITLE
Not an issue but a question about 10's action center on 11

### ISSUE THREAD

**Issue by newaccountpleaseignore** [bbmaster123/10SM#17]
i'm interested about the progress on the port of the 10's action center on 11. if there are any at all.

**Comment by bbmaster123** [comment]
ah yes, so to be completely honest the last time I worked on that was back on stable build .318 at which point I could get the action center to load reliably and perform 99% of its functions, but it still would hang on exit.

Since that time, MS changed a lot relating to the push notification service, so now restoring the action center means it also screws up toast notifications completely. Everything UWP ties into everything else, untangling it is like trying to untangle your spaghetti and meatballs lol

 Now I just moved to a new city literally a few days ago, and I'm still unpacking and settling in, but I do hope to get back into this soon and get a release of some kind out. Might still take me a while, I don't have a formal education in programming so I'm self teaching as I go. 
 
 have a great weekend! :)
